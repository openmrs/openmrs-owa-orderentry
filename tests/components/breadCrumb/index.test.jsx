import React from 'react';
import BreadCrumb from '../../../app/js/components/breadCrumb';

let props;
let mountedComponent;

const getComponent = () => {
  if (!mountedComponent) {
    mountedComponent = shallow(<BreadCrumb {...props} />);
  }
  return mountedComponent;
};

describe('Component: BreadCrumb', () => {
  beforeEach(() => {
    props = {
      patientId: 123,
      name:  'John.Doe',
      currentOrderTypeText: '',
    };
    mountedComponent = undefined;
  });

  it('renders properly', () => {
    const component = getComponent();
    expect(component).toMatchSnapshot();
  });
});
