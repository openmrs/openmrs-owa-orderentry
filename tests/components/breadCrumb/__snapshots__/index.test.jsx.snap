// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: BreadCrumb renders properly 1`] = `
<ul
  id="breadcrumbs"
>
  <li>
    <a
      href="../../"
    >
      <i
        className="icon-home small"
      />
    </a>
  </li>
  <li>
    <i
      className="icon-chevron-right link"
    />
    <a
      href="../../coreapps/clinicianfacing/patient.page?patientId=123"
    >
      John.Doe
    </a>
  </li>
  <li>
    <i
      className="icon-chevron-right link"
    />
  </li>
</ul>
`;
